wgpu-info/Cargo.toml

```diff
@@ -19,6 +19,7 @@ pico-args.workspace = true
 serde = { workspace = true, features = ["default"] }
 serde_json.workspace = true
 wgpu.workspace = true
+futures-lite.workspace = true
```

Comment:
Sorry I was just looking through this - we should use `pollster` here as that's what the examples use for blocking on futures.